SECURITY.md

```diff
@@ -0,0 +1,53 @@
+[English edition 🇺🇸](#Security-policy)
+
+<div dir="rtl">
+
+# سياسة الأمان
+لا يزال هذا المشروع قيد التطوير ، والهدف الأساسي في هذه المرحلة هو تنفيذ الميزات ولكننا أيضًا نأخذ الأمان على محمل الجد. يحدد هذا المستند السياسة الخاصة بكيفية الإبلاغ عن الثغرات الأمنية وتلقي التحديثات عند إصدار تصحيحات لتلك الثغرات.
+
+## الاصدارات المدعومة
+الاصدارات التي حالتها `✅` هي الاصدارات التي يهتم بأمانها فريق أمان أسس، والاصدارات التي حالتها `❌` هي الاصدارات التي لايهتم بأمانها فريق أمان أسس ( ينصح الانتقال الى الاصدارات التي يهتم بأمانها فريق أمان أسس ).
+
+|  الاصدار |  الحالة |
+|---------|-------|
+|         |       |
+
+## الإبلاغ عن ثغرة أمنية
+يأخذ فريق أمان أسس جميع الأخطاء الأمنية على محمل الجد. شكرًا لك على تحسين أمان تُرجمان. سنبذل قصارى جهدنا للاعتراف بمساهماتك.
+
+أبلغ عن الأخطاء الأمنية عن طريق إرسال بريد إلكتروني إلى فريق أمان أسس <security@aosus.org> سوف يتم مراجعة التقرير الخاص بك من قبل فريق أمان أسس، سوف يتم الرد عليك في اقرب مدة ممكنة وسوف يتم ارسال رد مفصل بعده في اقرب مدة ممكنة.
+
+## التعامل مع البلاغ
+يوجد نوعين من البلاغات الاول البلاغ الذي يتضمن حل للثغرة الامنية، والاخر الذي لايتضمن.
+
+### البلاغ الذي يتضمن حل للثغرة الامنية
+في حال كان بلاغك يتضمن خل للثغرة الامنية، سوف يراجع فريق أمان أسس الحل الخاص بك والتاكد من عدم وجود ثغرات اخرى وفي حال تم الموافقة على الحل سوف يقدم فريق أمان أسس تاريخ معين لك لعمل PR ليتم دمجه مباشرة الى المشروع وسوف يكون هذه مساهمة مباشرة في تُرجمان.
+
+### البلاغ الذي لا يتضمن حل للثغرة الامنية
+في حال كان بلاغك لا يتضمن حل للثغرة الامنية سوف يقوم فريق أمان أسس بمحاولة حل الثغرة الامنية والتاكد من عدم وجود ثغرات اخرى، وفي ال PR سوف يتم شكرك على المساهمة في أمان تُرجمان.
+
+</div>
+
+# Security policy
+This project is still in development, the primary goal at this point is to implement features but we also take security very seriously. This document sets out the policy on how vulnerabilities are reported and updates received when patches are released.
+
+## Supported versions
+The versions that have the status of `✅` are the versions whose safety the Aosus security team cares about, and the versions that have the status of `❌` are the versions that the Aosus security team doesn't care about (it is recommended to move to the versions whose safety the Aosus security team cares about).
```

Comment:
I think this should be changed to Maintained versions, instead of “versions that the Aosus development team cares about”
Also, Currently there is no dedicated security team, So I think you should put Aosus development team here

Also instead of emojis, we can just type maintained/unmaintained, and it would be very clear we wouldn't need this description.

Reply:
The Team name change suggestion is for all the document